modules/nf-core/gatk4/printsvevidence/main.nf

```diff
@@ -1,5 +1,5 @@
 process GATK4_PRINTSVEVIDENCE {
-    tag "${meta.id}"
+    tag "printsvevidence"
```

Comment:
should we keep meta.id here
```suggestion
    tag "${meta.id}"
```

Reply:
Since these files are a collection of all samples, you can't be sure that the meta will still contain an `id` field. so I think it would be better to keep it as it is?

Reply:
I think id is one of the stuff we can be sure is in the meta map.
even for a collection of sample, you can have `collection` as an id